packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -37,6 +41,23 @@ export async function processOutboundDocumentQueryResps({
   if (!(await isCQDirectEnabledForCx(cxId))) return interrupt(`CQ disabled for cx ${cxId}`);
 
   try {
+    const patient = await getPatientOrFail({ id: patientId, cxId: cxId });
+    const docQueryStartedAt = patient.data.documentQueryProgress?.startedAt;
+    const duration = diffFromNow(docQueryStartedAt);
+
+    analytics({
+      distinctId: cxId,
+      event: EventTypes.documentQuery,
+      properties: {
+        requestId,
+        patientId,
+        hie: MedicalDataSource.CAREQUALITY,
+        duration,
+        documentCount: results.length,
+      },
+      apiType: Product.medical,
```

Comment:
Can we make this the default value on `analytics`, or create an analytics function for this, please?

Reply:
I removed apiType seems like it was only necessary when we had DAPI